### ISSUE TITLE
Web3Modal types mismatch in @walletconnect/ethereum-provider

### ISSUE THREAD

**Issue by hoangbtmrk** [WalletConnect/walletconnect-monorepo#2295]
**Describe the bug**
I install ethereum-provider lib and serve app but get errors in EthereumProvider.d.ts file, I find the cause is in 2.3.0 release of web3modal, they removed: explorerAllowList and explorerDenyList options in favour of explorerRecommendedWalletIds and explorerExcludedWalletIds, but in EthereumProvider.d.ts file still use explorerAllowList & explorerDenyList
here is the log of errors

```
Error: node_modules/@walletconnect/ethereum-provider/dist/types/EthereumProvider.d.ts:7:60 - error TS2344: Type '"mobileWallets" | "desktopWallets" | "walletImages" | "chainImages" | "tokenImages" | "enableExplorer" | "termsOfServiceUrl" | "privacyPolicyUrl" | "themeMode" | "themeVariables" | "explorerAllowList" | "explorerDenyList"' does not satisfy the constraint '"projectId" | "walletConnectVersion" | "standaloneChains" | "defaultChain" | "mobileWallets" | "desktopWallets" | "walletImages" | "chainImages" | "tokenImages" | "tokenContracts" | ... 7 more ... | keyof ThemeCtrlState'.
  Type '"explorerAllowList"' is not assignable to type '"projectId" | "walletConnectVersion" | "standaloneChains" | "defaultChain" | "mobileWallets" | "desktopWallets" | "walletImages" | "chainImages" | "tokenImages" | "tokenContracts" | ... 7 more ... | keyof ThemeCtrlState'.

7 export declare type QrModalOptions = Pick<Web3ModalConfig, "themeMode" | "themeVariables" | "chainImages" | "desktopWallets" | "enableExplorer" | "explorerAllowList" | "explorerDenyList" | "mobileWallets" | "privacyPolicyUrl" | "termsOfServiceUrl" | "tokenImages" | "walletImages">;
```

**SDK Version**
 - Client: @walletconnect/ethereum-provider
 - Version [2.7.2]

**To Reproduce**
1. ng new web3-app
2. npm i @walletconnect/ethereum-provider @web3modal/standalone
3. import { EthereumProvider } from "@walletconnect/ethereum-provider";
4. ng serve and see errors

**Expected behavior**
No error in EthereumProvider.d.ts file

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
I try to remove explorerAllowList, explorerDenyList and it worked

**Comment by hoangbtmrk** [comment]
Has anyone investigated this issue? I'm still stuck here

**Comment by layinka** [comment]
+1 for this

**Comment by ganchoradkov** [comment]
Hey. we've just released 2.7.4 and the issue should be fixed
